samples/demos/end_to_end_tank/main.py

```diff
@@ -455,16 +478,19 @@ def upload_sensor_data():
     # Calculate the new tank level.
     calculate_tank_level()
     tank_level_f = "{:.2f}".format(tank_level)
+    tank_temperature_f = "{:.2f}".format(get_tank_temperature())
 
     # print debug traces.
     print("- Sending sensor values to DRM:")
     print("  - Tank level: {} %".format(tank_level_f))
     print("  - Tank valve: {}".format("Open" if tank_valve_open else "Closed"))
+    print("  - Tank temperature: {} C".format(tank_temperature_f))
 
     # Upload the samples to DRM.
     data = cloud.DataPoints()
     data.add(DATAPOINT_LEVEL, tank_level_f)
```

Comment:
Might it make sense to pass `units` here? E.g. `data.add(DATAPOINT_LEVEL, tank_level_f, units="percent")`.

Reply:
And include `units="C"` for the temperature as well?